hack/00-osd-managed-prometheus-exporter-machine-api.selectorsyncset.yaml.tmpl

```diff
@@ -0,0 +1,234 @@
+apiVersion: v1
```

Comment:
Isn't that file generated dynamically? Does it make sense to keep it in the repo?

Reply:
The other exporters seem to have it commited?

Reply:
This file is what saasherder pulls and uses to configure things on hive.  It has to exist and can't be built dynamically since the jenkins job pulls the raw file using curl or wget or something.